src/services/inplayer.service.ts

```diff
@@ -0,0 +1,35 @@
+export const getBaseUrl = (sandbox: boolean) => (sandbox ? 'https://services-daily.inplayer.com' : 'https://services.inplayer.com');
```

Comment:
If all the methods are updated to be used from the sdk, this file is no longer needed.